Monika After Story/game/script-topics.rpy

```diff
@@ -4827,35 +4827,35 @@ init 5 python:
     monika_random_topics.append('monika_colorblindness')
 	
 label monika_colorblindness:
-	m "Hey [player], I've been reading up a bit on color blindness."
-	m "I just can't imagine how it must be like!"
-	m "To never be able to see the roaring sun by its true colors or its gentle sunset!"
-	m "I really feel bad for those poor people."
-	m "Most of them are male too, since it's quite rare for a woman to have a color blind deficit of some sort."
-	m "Gosh, genetics are super weird, aren't they?"
-	m "Why don't we do a little experiment?"
-	menu:
-		m "What are the color of my eyes?"
-		"Green":
-			m "Ahaha, you're right!"
-			m "Well, at least that means you're not color blind."
-			m "Unless you lied to me, but why would you?"
-			m "Ah, in that case you, maybe just didn't want to worry me."
-			m "I'm always willing to deal with any of your worries [player]!"
-		"Gray":
-			m "Gosh, we might have an issue here."
-			m "If you see everything on a scale of greys, that could mean you're completely color blind!”
-			m "Unless you just have a super weird black and white monitor.
-			m "But take care of yourself."
+     m "Hey [player], I've been reading up a bit on color blindness."
+     m "I just can't imagine how it must be like!"
+     m "To never be able to see the roaring sun by its true colors or its gentle sunset!"
+     m "I really feel bad for those poor people."
+     m "Most of them are male too, since it's quite rare for a woman to have a color blind deficit of some sort."
+     m "Gosh, genetics are super weird, aren't they?"
+     m "Why don't we do a little experiment?"
+     menu:
+	m "What are the color of my eyes?"
+	"Green":
+		m "Ahaha, you're right!"
+		m "Well, at least that means you're not color blind."
+		m "Unless you lied to me, but why would you?"
+		m "Ah, in that case you, maybe just didn't want to worry me."
+		m "I'm always willing to deal with any of your worries [player]!"
+	"Gray":
+		m "Gosh, we might have an issue here."
+		m "If you see everything on a scale of greys, that could mean you're completely color blind!”
```

Comment:
The quote at the end of this line is a bad one. Please replace with regular double quote.